.travis.yml

```diff
@@ -2,19 +2,19 @@ language: php
 
 sudo: false
 
-php: [5.4, 5.5, 5.6, 7.0, 7.1, hhvm]
+php: [5.4, 5.5, 5.6, 7.0, 7.1, 7.2]
 
 matrix:
   include:
     - php: 5.3
```

Comment:
Not sure what's with this entry. It's not in the list above. Should we remove this?

Reply:
Nope it should stay:

* in `php` setting we list the PHP version **without** custom configuration
* in `matrix` setting we specify PHP versions **with** custom configuration

Reply:
Thanks for explanation!